### PR TITLE
[PR] Warning team switch + mail submit + 

### PR DESCRIPTION
**Pull request by mathisfchx**
### Description
Warning when team switch + sending emails
Refactor front for relations
+ bug fixs

See attached issues in the commits

### CODE REVIEW

File: Netw4ppl/app/src/main/res/values-fr/strings.xml [hunk 1]
```diff
@@ -79,4 +79,12 @@
     <string name="no_bdv_message">Demandez à votre administrateur d\'ajouter un champ Best Descriptive Value à votre équipe. Actuellement, l\'application est inutilisable.</string>
     <string name="no_bdv_confirmation">Quitter</string>
     <string name="toast_save_failed">L\'enregistrement de la nouvelle personne a échoué</string>
+    <string name="team_change_title">On vous a attribué une nouvelle équipe !</string>
+    <string name="team_change_message">Cela implique un nouveau formulaire pour créer des personnes, toutes les données actuellement non synchronisés stockées dans votre application seront perdues. Vous pouvez ci dessous les sauvegarder en les envoyant par mail.</string>
```
Comment (lduf) [inline Netw4ppl/app/src/main/res/values-fr/strings.xml]:
```suggestion
    <string name="team_change_message">Cela implique un nouveau formulaire pour créer des personnes, toutes les données actuellement non synchronisées stockées dans votre application seront perdues. Vous pouvez ci dessous les sauvegarder en les envoyant par email.</string>
```

File: Netw4ppl/app/src/main/res/values-fr/strings.xml [hunk 2]
```diff
@@ -79,4 +79,12 @@
...
+    <string name="team_change_send_mail">Envoyer mes anciennes données par mail</string>
```
Comment (lduf) [inline Netw4ppl/app/src/main/res/values-fr/strings.xml]:
```suggestion
    <string name="team_change_send_mail">Envoyer mes anciennes données par email</string>
```

File: Netw4ppl/app/src/main/res/values-fr/strings.xml [hunk 3]
```diff
@@ -79,4 +79,12 @@
...
+    <string name="team_change_confirmation">Continuer (Mes anciennes données seront perdues)</string>
+    <string name="change_team_back_title">Faites attention !</string>
+    <string name="change_team_back_message">Avez vous envoyé vos datas par mail ?</string>
```
Comment (lduf) [inline Netw4ppl/app/src/main/res/values-fr/strings.xml]:
```suggestion
    <string name="change_team_back_message">Avez vous envoyé vos données par email ?</string>
```

File: Netw4ppl/app/src/main/res/values-fr/strings.xml [hunk 4]
```diff
@@ -79,4 +79,12 @@
...
+    <string name="change_team_back_yes">Oui, supprimer mes anciennes données et changer d\'équipe</string>
+    <string name="change_team_back_send_mail">Non, envoyer mes anciennes données par mail</string>
```
Comment (lduf) [inline Netw4ppl/app/src/main/res/values-fr/strings.xml]:
```suggestion
    <string name="change_team_back_send_mail">Non, envoyer mes anciennes données par email</string>
```

File: Netw4ppl/app/src/main/res/values/strings.xml [hunk 5]
```diff
@@ -149,4 +149,16 @@
 
     <string name="toast_save_failed">Saving of the new Person has failed</string>
 
+    <string name="crew_id_path" translatable="false">/crew_id.json</string>
+
+    <string name="team_change_title">Your Team has changed !</string>
+    <string name="team_change_message">This involves a new form for creating people, all currently unsynchronised data stored in your application will be lost. You can save them below by sending them by email.</string>
+    <string name="team_change_send_mail">Upload old data by mail</string>
```
Comment (lduf) [inline Netw4ppl/app/src/main/res/values/strings.xml]:
```suggestion
    <string name="team_change_send_mail">Upload old data by email</string>
```

File: Netw4ppl/app/src/main/res/values/strings.xml [hunk 6]
```diff
@@ -149,4 +149,16 @@
...
+    <string name="team_change_confirmation">Continue (old data will be lost)</string>
+
+    <string name="change_team_back_title">Be careful!</string>
+    <string name="change_team_back_message">Did you send your data by email?</string>
+    <string name="change_team_back_yes">Yes, delete old data + change team</string>
```
Comment (lduf) [inline Netw4ppl/app/src/main/res/values/strings.xml:161]:
```suggestion
    <string name="change_team_back_yes">Yes, delete old data & change team</string>
```

File: Netw4ppl/app/src/main/java/netw4ppl/ines/DisplayDetailsRelationActivity.java [hunk 7]
```diff
@@ -54,13 +63,28 @@ protected void onCreate(Bundle savedInstanceState) {
             index_relation = 0;
         }
 
-        mTextViewFromFullname = (TextView) findViewById(R.id.relation_from_fullname);
-        mTextViewRelationType = (TextView) findViewById(R.id.relation_type);
-        mTextViewToFullname = (TextView) findViewById(R.id.relation_to_fullname);
-        mTextViewComments = (TextView) findViewById(R.id.relation_comment);
+        mFromTitle = (com.google.android.material.textfield.TextInputLayout) findViewById(R.id.relation_from_title);
```
Comment (lduf) [inline Netw4ppl/app/src/main/java/netw4ppl/ines/DisplayDetailsRelationActivity.java:66]: #134 
-> use an array ?

File: Netw4ppl/app/src/main/java/netw4ppl/ines/MainActivity.java [hunk 8]
```diff
@@ -86,6 +91,13 @@ protected void onCreate(Bundle savedInstanceState) {
             Intent intent = new Intent(MainActivity.this, SettingsActivity.class);
             startActivity(intent);
         });
+        mInfo.setOnClickListener(v ->{
+                Intent intent = new Intent();
+                intent.setAction(Intent.ACTION_VIEW);
+                intent.addCategory(Intent.CATEGORY_BROWSABLE);
+                intent.setData(Uri.parse("https://user-doc.netw4ppl.tech"));
```
Comment (lduf) [inline Netw4ppl/app/src/main/java/netw4ppl/ines/MainActivity.java:98]: Export to xml ?

File: Netw4ppl/app/src/main/java/netw4ppl/ines/utils/SubmitData.java [hunk 9]
```diff
@@ -195,17 +237,23 @@ public static void sendByEmail(Context context, String filePath) {
...
         File[] arrFiles = directory.listFiles();
+        //remove the files in arrFiles containing "id" and "synced" in their path
+        if (Build.VERSION.SDK_INT >= Build.VERSION_CODES.N) {
+            arrFiles = Arrays.stream(arrFiles).filter(f -> !f.getPath().contains("id") && !f.getPath().contains("synced")).toArray(File[]::new);
+        }
 
         // for every file present in the directory, add it in the attachments list
         if (arrFiles != null) {
             for (File f : arrFiles){
+                System.out.println(f.getAbsolutePath());
```
Comment (lduf) [inline Netw4ppl/app/src/main/java/netw4ppl/ines/utils/SubmitData.java:256]:
```suggestion
               
```

**Review by lduf — CHANGES_REQUESTED** [review]

File: Netw4ppl/app/src/main/java/netw4ppl/ines/MainActivity.java (hunk 8 above)
Comment (lduf) [reply Netw4ppl/app/src/main/java/netw4ppl/ines/MainActivity.java:98]: To do in #134

**Review by lduf — DISMISSED** [review]
🌍

File: Netw4ppl/app/src/main/res/values/strings.xml [hunk 10]
```diff
@@ -149,4 +149,16 @@
...
+    <string name="crew_id_path" translatable="false">/crew_id.json</string>
+
+    <string name="team_change_title">Your Team has changed !</string>
+    <string name="team_change_message">This involves a new form for creating people, all currently unsynchronised data stored in your application will be lost. You can save them below by sending them by email.</string>
+    <string name="team_change_send_mail">Upload old data by email</string>
+    <string name="team_change_confirmation">Continue (old data will be lost)</string>
+
+    <string name="change_team_back_title">Be careful!</string>
+    <string name="change_team_back_message">Did you send your data by email?</string>
+    <string name="change_team_back_yes">Yes, delete old data & change team</string>
```
Comment (mathisfchx) [inline Netw4ppl/app/src/main/res/values/strings.xml]:
```suggestion
    <string name="change_team_back_yes">Yes, delete old data + change team</string>
```

**Review by lduf — APPROVED** [review]
🌍

**Pull request merged**